src/libOpenImageIO/imagebuf.cpp

```diff
@@ -677,7 +677,7 @@ ImageBufImpl::error(string_view message) const
     // a single newline.
     if (m_err.size() && m_err.back() != '\n')
         m_err += '\n';
-    m_err += message;
+    m_err += std::string(message);
```

Comment:
What was the error message that led you to this change? I think it's fine, but I'm curious why it was necessary given that it has been accepted just fine by all the compilers we test for a long time.